apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -1955,6 +1955,7 @@ describe('Meteor.methods', function () {
 
 	describe('[@updateMessage]', () => {
 		let rid = false;
+		let roomName = false;
```

Comment:
Any reason to define `false` here? 🤔

Reply:
No, that's just me copy+pasting :see_no_evil: